tools/mo/openvino/tools/mo/front/tf/reduce_ext.py

```diff
@@ -17,6 +18,16 @@ def extract(cls, node: Node):
         return cls.enabled
 
 
+class AnyExtractor(FrontExtractorOp):
+    op = 'Any'
+    enabled = True
+
+    @classmethod
+    def extract(cls, node):
+        ReduceLogicalOr.update_node_stat(node, {'keep_dims':  node.pb.attr['keep_dims'].b})
```

Comment:
Shouldn't we check if `keep_dims` in attributes?
```
        keep_dims = False
        if 'keep_dims' in node.pb.attr:
            keep_dims = node.pb.attr['keep_dims'].b
```
This is according to the specification: https://www.tensorflow.org/api_docs/python/tf/raw_ops/Any

Reply:
It's not necessary, because `keep_dims` is always present in the pb, even if user didn't specify it TF puts default `False` into protobuf. We have exactly same extractor for all tensorflow logical reduce ops
https://github.com/openvinotoolkit/openvino/blob/0c9abf43a91fbe2dc95974b6cb2651bd883d1b12/tools/mo/openvino/tools/mo/front/tf/reduce_ext.py#L19-L47

Also i double checked locally. Here it is, i didn't specify `keeps_dims` when created the op and it's still in the pb
![image](https://user-images.githubusercontent.com/5703039/215065035-7b810d5c-5075-4381-a2d5-4533dfbca883.png)

![image](https://user-images.githubusercontent.com/5703039/215065052-f849b23f-3dba-4442-bbd6-c7b7a599e13e.png)